src/utils.ts

```diff
@@ -125,6 +125,10 @@ export namespace ControllerUtils {
 		return res.status(422).send({ message });
 	};
 
+	export const unableToGenerateId = (res: Response, message: string): any => {
```

Comment:
don't declare the return type as `any`, if you are unsure what return type you want then let the function infer it from your return statement.